crates/mysten-common/Cargo.toml

```diff
@@ -6,8 +6,15 @@ version = "0.1.0"
 edition = "2021"
 publish = false
 
+[lints]
+workspace = true
+
 [dependencies]
+rand.workspace = true
+antithesis_sdk.workspace = true
```

Comment:
Do we want this to be gated by some config so that it doesn't ship in release builds?

Reply:
I thought I answered the same comment from someone else but maybe not.

antithesis-sdk is designed to work in release builds. you can collect information about things like reachability assertions from live servers. Not sure if it will be useful yet but I see no reason to do extra work to avoid shipping it.

https://antithesis.com/docs/using_antithesis/sdk/rust/##sdk-runtime-behavior